### ISSUE TITLE
3DS QR code using wrong IP address

### ISSUE THREAD

**Issue by the0remora** [danmons/retronas#159]
I ran into an issue with the /opt/retronas/scripts/3ds_qr.sh file. I assign addresses via DHCP and usually have PfSense set up with a Static Assignment. At some point during testing I think I used the WiFi on my Pi4 instead of the LAN and it got the address 172.17.*.* it wrote that into the 3ds_qr.sh file and never updated it (LAN uses 172.19.*.*). Could some way be implemented RetroNAS to force an update either periodically or during reboot or as part of one of the update scripts?

TLDR: RetroNAS got an IP address during testing, that address has changed but the old one is still used by scripts.

**Comment by sairuk** [comment]
Not really an issue since the script worked as defined, am converting to discussion to continue with enhancement request conversation
